AdyenComponents/Apple Pay/ApplePayComponentExtensions.swift

```diff
@@ -115,6 +115,18 @@ extension ApplePayComponent {
             paymentRequest.billingContact = billingContact
             return paymentRequest
         }
+
+        public mutating func update(amount: Amount, localeIdentifier: String?) {
+            var newItems = summaryItems
+            guard let lastItem = newItems.last else { return }
+
+            newItems = newItems.dropLast()
+            let newAmount = AmountFormatter.decimalAmount(amount.value,
```

Comment:
Can't we use the new amount object as is? Why do we need to specify localeIdentifier explicitly?

Reply:
hm.. this is good question which one we should use here 🤔

Reply:
I think `localeIdentifier` on amount is probably always empty, since this is `remainingAmount`, that we receive from backend 🤔